### ISSUE TITLE
Make sure that data links are on appropriate pages

### ISSUE THREAD

**Issue by plantfansam** [sunlightlabs/mptindicators#86]
Refers to .zip file with all data.

**Comment by cweber** [comment]
Honestly...I can't remember where these are supposed to be and will likely come up at walk through regardless. Waiting on this too.

**Comment by plantfansam** [comment]
closing in favor of #103
